content/blog/running-self-hosted-agents-in-azure-devops-using-an-azure-ad-access-token.md

```diff
@@ -0,0 +1,126 @@
+---
+id: blog
+title: Running Self Hosted Agents in Azure Devops using an Azure AD Access Token
+description: Running Self Hosted Agents used to only be possible using Personal Access Tokens, which are a chore to rotate, bound to a single person and often way overprovisioned, making them a security risk. Is there a better way? Let's find out.
```

Comment:
Please add a `summary` here as well, so that it doesn't just take the first bit of the actual blog on the blog index page 😉